src/transformers/models/auto/modeling_auto.py

```diff
@@ -19,8 +19,6 @@
 from collections import OrderedDict
 
 from ...utils import logging
-
-# Add modeling imports here
```

Comment:
It's always moving around with the template. If you could put it back on top it would be great (and we should fix that some day).